backend/src/main/resources/static/css/admin.css

```
@@ -1,65 +1,5 @@
-.page-info { font-size: 0.85rem; color: #6b7280; }
-.page-btn {
-    padding: 0.3rem 0.65rem; border: 1px solid #d1d5db; border-radius: 5px;
-    background: #fff; font-size: 0.8rem; color: #374151; text-decoration: none;
-}
-.page-btn:hover { background: #f9fafb; }
-.page-current {
-    padding: 0.3rem 0.65rem; border: 1px solid #6366f1; border-radius: 5px;
-    background: #6366f1; font-size: 0.8rem; color: #fff; font-weight: 700;
-}
-.page-ellipsis { font-size: 0.8rem; color: #9ca3af; }
+@import url('https://fonts.googleapis.com/css2?family=Noto+Sans+KR:wght@400;500;600;700&family=JetBrains+Mono:wght@400;500&display=swap');
```

Stylelint (17.6.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Noto+Sans+KR:wght@400;500;600;700&family=JetBrains+Mono:wght@400;500&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Noto+Sans+KR:wght@400;500;600;700&family=JetBrains+Mono:wght@400;500&display=swap'" (import-notation)

(import-notation)